javascript-bank/bank.js

```diff
@@ -0,0 +1,32 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  var newAccount = new Account(this.nextAccountNumber, holder);
+  if (balance > 0 && (Number.isInteger(balance) === true)) {
+    newAccount.deposit(balance);
+    this.accounts.push(newAccount);
+    this.nextAccountNumber++;
+    return newAccount.number;
+  }
+  return null;
+};
+
+Bank.prototype.getAccount = function (number) {
+  if (number < this.accounts.length + 1) {
+    return this.accounts[number - 1];
```

Comment:
This is not the right way to do this, you should **_never_** use ids for math. You need to loop over the list of accounts, and find the one with the matching number to the number that was requested.